docs/development/reviewing-patches.rst

```diff
@@ -34,6 +34,47 @@ These are small things that are not caught by the automated style checkers.
 * Does a variable need a better name?
 * Should this be a keyword argument?
 
+Testing branches remotely
+-------------------------
+
+It can be helpful to check out a branch with a PR on it, in order to test
+how the changes in it work on your machine. To do so, first pull
+down the PR branch by running the following commands:
+
+.. code-block:: console
+
+  $ git checkout master
+  $ git fetch upstream pull/ID/head:BRANCHNAME
+
+In this example, upstream refers to the Warehouse repo on github.
+ID refers to the sequence of digits after the pull request's title.
+BRANCHNAME creates a new branch on your machine which has the pull request
+on it. If you create a branch with the name BRANCHNAME before pulling down
+the PR branch, this command will fail.
```

Comment:
> If you create a branch with the name

Suggested rewording: If you create a branch in your local repository with the name

I suggest you put ``backticks`` around BRANCHNAME to make it visually distinct.